frontend/src/metabase-lib/queries/StructuredQuery.ts

```diff
@@ -386,7 +385,6 @@ class StructuredQueryInner extends AtomicQuery {
       .cleanFilters()
       .cleanAggregations()
       .cleanBreakouts()
-      .cleanSorts()
```

Comment:
Now order-bys can only be added from the notebook editor via MLv2. MLv2 guarantees that a query is valid at any given point of time, so we can skip the clean step for them.